pine64/pinebook-pro/firmware/ap6256-firmware/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitLab
+, fetchurl
+, runCommandNoCC
+}:
+
+let
+  src = fetchFromGitLab {
+    domain = "gitlab.manjaro.org";
+    owner = "manjaro-arm";
+    repo = "packages/community/ap6256-firmware";
```

Comment:
The original firmware was deleted but I recovered the firmware from a fork and mirrored it here: https://github.com/nix-community/rkwifibt Can we use this? It looks a bit less random than pulling firmware files from some gitlab mirror.

Reply:
Nice idea, I'm changing it.

Reply:
@Mic92 there is a problem, In the repo you mirrored I can't find a needed file (`brcmfmac43456-sdio.clm_blob`). I found some repos on github that provides it but I can't find the better/more upstream one. What do you suggest?